grakn-graph/src/test/java/ai/grakn/generator/TypeNames.java

```diff
@@ -50,14 +50,21 @@ public TypeName generateFromGraph() {
             return withImplicitConceptsVisible(graph(), graph -> {
                 TypeName name;
 
+                int attempts = 0;
                 do {
-                    name = randomName();
+                    // After a certain number of attempts, generate truly random strings instead
```

Comment:
This scares me. We either dumb luck our way into something readable or we smash against a wall give up and go with something unreadable?